connector/src/main/java/org/jboss/as/connector/services/driver/InstalledDriver.java

```diff
@@ -88,7 +86,7 @@ public InstalledDriver(final String deploymentUnitName, final String driverClass
      * @return the module name, or {@code null} if {@link #isFromDeployment()}
```

Comment:
This should now be:

```
@return the canonical module name, or {@code null} if {@link #isFromDeployment()} returns {@code true}
```

And then this PR needs to be sure only canonical strings are passed to the constructor. (Which probably it does; I haven't double-checked as I write this.)

The constructor javadoc for the moduleName parameter should declare that the moduleName must be canonical.

Reply:
Thanks! @bstansberry review it once.